lib/components/Input/InputGroup/Button/index.tsx

```diff
@@ -0,0 +1,70 @@
+import { forwardRef, useEffect, useState } from "react";
+import Input, { InputProps } from "@components/Input/base";
+import { Button, TVariant } from "@components/Button";
```

Comment:
Need to refactor this in other PR: 
```suggestion
import { Button, TButtonVariant } from "@components/Button";
```